### PR TITLE
new filter by submission_status

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3306306&stafilter=NotComplete&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

File: app/Http/Controllers/Apis/Protected/Summit/Strategies/events/RetrieveSummitEventsStrategy.php [hunk 1]
```diff
@@ -248,6 +249,7 @@ protected function getFilterValidatorRules(): array
             'streaming_url' => 'sometimes|string',
             'etherpad_link' => 'sometimes|string',
             'streaming_type' => 'sometimes|string|in:VOD,LIVE',
+            'submission_status' => 'sometimes|string',
```
Comment (smarcet) [inline app/Http/Controllers/Apis/Protected/Summit/Strategies/events/RetrieveSummitEventsStrategy.php:252]: we need to define possible values here
bc on db are NULL or Received
so i would said in:Received,NonReceived

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 2]
```diff
@@ -236,6 +236,7 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
             'abstract' => 'e.abstract:json_string',
             'level' => 'e.level:json_string',
             'status' => 'p.status:json_string',
+            'submission_status' => 'p.status:json_string',
```
[the remark this answers is not in the document]
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php]: done

File: app/Http/Controllers/Apis/Protected/Summit/Strategies/events/RetrieveSummitEventsStrategy.php (hunk 1 above)
Comment (romanetar) [reply app/Http/Controllers/Apis/Protected/Summit/Strategies/events/RetrieveSummitEventsStrategy.php]: done

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**